tools/perf/ucc_pt_coll_reduce.cc

```diff
@@ -0,0 +1,77 @@
+#include "ucc_pt_coll.h"
+#include "ucc_perftest.h"
+#include <ucc/api/ucc.h>
+#include <utils/ucc_math.h>
+#include <utils/ucc_coll_utils.h>
+
+ucc_pt_coll_reduce::ucc_pt_coll_reduce(ucc_datatype_t dt, ucc_memory_type mt,
+                        ucc_reduction_op_t op, bool is_inplace, bool is_root)
+{
+    has_inplace_   = true;
+    has_reduction_ = true;
+    has_range_     = true;
+    has_bw_        = true;
+    is_root_       = is_root;
```

Comment:
why not just do is_root_ = rank==0, and then you won't need to change all perf files to add is_root